apps/api/src/server/repositories/project-update.respository.js

```diff
@@ -8,26 +8,59 @@ import BackOfficeAppError from '#utils/app-error.js';
  */
 
 /**
- * List project updates for a particular case
+ * @typedef {Object} PaginationOptions
+ * @property {number} page
+ * @property {number} pageSize
+ */
+
+/**
+ * @typedef {Object} ListProjectUpdatesOptions
+ * @property {number} page
+ * @property {number} pageSize
+ * @property {import('@prisma/client').Prisma.ProjectUpdateOrderByWithRelationInput} [orderBy]
+ * @property {number} [caseId] - filter option
+ * @property {string} [status] - filter option
+ * @property {Date} [publishedBefore] - filter option (datePublished < publishedBefore)
+ * @property {boolean} [sentToSubscribers] - filter option
+ */
+
+/**
+ * List project updates for a particular case, with filter options such as by case
  *
- * @param {number} caseId
- * @param {number} pageNumber
- * @param {number} pageSize
- * @param {import('@prisma/client').Prisma.ProjectUpdateOrderByWithRelationInput} [orderBy]
+ * @param {ListProjectUpdatesOptions} opts
  * @returns {Promise<{count: number, items: import('@prisma/client').ProjectUpdate[]}>}
  */
-export async function listProjectUpdates(caseId, pageNumber, pageSize, orderBy) {
-	const where = {
-		caseId
-	};
+export async function listProjectUpdates({
+	caseId,
+	page,
+	pageSize,
+	orderBy,
+	status,
+	publishedBefore,
+	sentToSubscribers
+}) {
+	/** @type {import('@prisma/client').Prisma.ProjectUpdateWhereInput} */
+	const where = {};
+	if (caseId) {
+		where.caseId = caseId;
+	}
+	if (status) {
+		where.status = status;
+	}
+	if (publishedBefore) {
+		where.datePublished = { lt: publishedBefore };
+	}
+	if (sentToSubscribers) {
```

Comment:
Should this have three possible values? True, False and Empty? 

Right now if this is passed in as false, the filter will never be applied - so both will be returned.

Reply:
Good catch, needs a `typeof sentToSubscribers !== 'undefined'` check